src/templates/case-study.jsx

```diff
@@ -89,3 +84,14 @@ export const query = graphql`
 `;
 
 export default CaseStudyTemplate;
+
+export const Head = ({
+  data: {
+    mdx: { frontmatter },
+  },
+}) => (
+  <SEO
+    title={SEO_DATA.caseStudy({ title: frontmatter.title }).title}
+    description={frontmatter.description}
```

Comment:
```suggestion
  <SEO
    {...SEO_DATA.caseStudy({ title, description })}
```

Reply:
Or it can be 
```suggestion
  <SEO
    title={SEO_DATA.caseStudy({ title }).title}
    description={SEO_DATA.caseStudy({ description }).description}
```